lib/src/widgets/chatui_textfield.dart

```diff
@@ -88,15 +90,36 @@ class _ChatUITextFieldState extends State<ChatUITextField> {
             BorderRadius.circular(textFieldBorderRadius),
       );
 
+  StreamController<String> typingStreamController = StreamController();
+
+  ValueNotifier<ComposingStatus> composingStatus =
+      ValueNotifier(ComposingStatus.composed);
+
   @override
   void initState() {
+    // In init function
+    attachListeners();
+    typingStreamController.stream
+        .debounce(widget
+                .sendMessageConfig?.textFieldConfig?.compositionThresholdTime ??
+            const Duration(seconds: 1))
```

Comment:
Could you please add this default duration in a model and make `compositionThresholdTime` non-nullable?

Reply:
I can do that too, but I'm having some queries .

1. Why to do so ?
2.  In which model class should i add this.

Thanks and regards.

Reply:
In actuality, I was checking PR. I added this comment for reference because parallel I am testing this too locally. But, now I think this isn't required.

Reply:
In actuality, I was checking PR. I added this comment for reference because parallel I am testing this too locally. But, now I think this isn't required.

Reply:
I will update you once I review this PR. Thanks!!

Reply:
Alright regards 😀!